common/constants/default.go

```diff
@@ -54,8 +54,8 @@ const (
 	DefaultVolumeStatsAggPeriod        = time.Minute
 	DefaultTunnelPort                  = 10004
 	DefaultClusterDomain               = "cluster.local"
-
-	CurrentSupportK8sVersion = "v1.26.7"
+	CurrentSupportK8sVersion           = "v1.26.7"
+	DefaultQuicPort                    = 10001
```

Comment:
A small suggestion: It would be more appropriate to separate this constant into its own section, like: 
// EdgeHub
DefaultQuicPort                    = 10001

Reply:
ok ，I have made it.